src/ExternalApiExamples/appsettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "TokenProvider": {
+    "AuthorizationScope": "",
```

Comment:
We could probably set this. Only problem is it will be different for Prod and Review.

Reply:
I would leave it up for the tester. Let him do the most of the configuration on his own.